crates/goose-mcp/src/google_drive/mod.rs

```diff
@@ -1343,68 +1357,105 @@ impl GoogleDriveRouter {
-            .await;
+            .unwrap_or(Ok(20))?;
+
+        if let Some(comment) = comment_id {
+            // Use the get comment method to read a single comment
+            let result = self
+                .drive
+                .comments()
+                .get(file_id, comment)
+                .param("fields", "*")
+                .clear_scopes()
+                .add_scope(GOOGLE_DRIVE_SCOPES)
+                .doit()
+                .await;
+
+            match result {
+                Err(e) => Err(ToolError::ExecutionError(format!(
+                    "Failed to execute google drive comment read, {}.",
+                    e
+                ))),
+                Ok(r) => {
+                    let content = format!(
+                        "Author:{:?} Quoted File Content: {:?} Content: {} Replies: {:?} (created time: {}) (modified time: {})(anchor: {}) (resolved: {}) (id: {})",
+                        r.1.author.unwrap_or_default(),
+                        r.1.quoted_file_content.unwrap_or_default(),
+                        r.1.content.unwrap_or_default(),
+                        r.1.replies.unwrap_or_default(),
+                        r.1.created_time.unwrap_or_default(),
+                        r.1.modified_time.unwrap_or_default(),
+                        r.1.anchor.unwrap_or_default(),
+                        r.1.resolved.unwrap_or_default(),
+                        r.1.id.unwrap_or_default()
+                    );
 
-        match result {
-            Err(e) => Err(ToolError::ExecutionError(format!(
-                "Failed to execute google drive comment list, {}.",
-                e
-            ))),
-            Ok(r) => {
-                let content =
-                    r.1.comments
-                        .map(|comments| {
-                            comments.into_iter().map(|c| {
-                                format!(
-                                    "Author:{:?} Content: {} (created time: {}) (modified time: {})(anchor: {}) (resolved: {}) (id: {})",
-                                    c.author.unwrap_or_default(),
-                                    c.content.unwrap_or_default(),
-                                    c.created_time.unwrap_or_default(),
-                                    c.modified_time.unwrap_or_default(),
-                                    c.anchor.unwrap_or_default(),
-                                    c.resolved.unwrap_or_default(),
-                                    c.id.unwrap_or_default()
-                                )
+                    Ok(vec![Content::text(content.to_string())])
+                }
+            }
+        } else {
+            let result = self
+                .drive
+                .comments()
+                .list(file_id)
+                .page_size(page_size)
+                .param(
+                    "fields",
+                    "comments(author, content, createdTime, modifiedTime, id, resolved)",
```

Comment:
is it possible to also pull replies here? i can imagine users wanting to just pull all comments + replies in a single call to have goose synthesize everything